dist/svg/icon/icon-star-empty.svg

```diff
@@ -1,5 +0,0 @@
-<svg viewbox="0 0 22 22" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I was trying to figure that out as well, something about the build isn't adding it.  I'll dig some more

Reply:
@ArtBlue so the reason these star svg icons are gone is because in the src svg folder there are now two versions for each a -small and -large.  Guess a previous build didn't remove them before